src/Services/PatientService.php

```diff
@@ -52,16 +51,16 @@ public function validate($patient, $context, $id = null)
         
         $this->validator->context('insert', function (Validator $context) {
             $context->required('first_name')->lengthBetween(2, 30);
```

Comment:
Whats diff btw `fname` and `first_name` 😕

Reply:
There isn't a first_name in patient data. Generally that syntax is used for user etc.  So, we need to be careful that array pass ins follow table column naming conventions.
Probably need to fix this Yash....

Reply:
Do i just need to remove this ?

Reply:
As long as we still require fname for insert. ie first_name is fname etc

Reply:
Ohh so we need to show a clear message that `first name` is required because `fname` isn't so clear. Right?

Reply:
depends how granular you want to report error Yash. I think first name is fine for both if fname data element not passed in or value is missing.

Reply:
Ok for that we have already given the name of every index to Validator. That will be consistent and will do a better job.

Reply:
okay, lets fix and get this in so I can test while working on other stuff! fname is our guy...

Reply:
Done 👍